tools/SetupFlow/DevHome.SetupFlow/Controls/PackageDetailsSettingsCard.xaml.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using CommunityToolkit.WinUI.Controls;
+
+namespace DevHome.SetupFlow.Controls;
+public sealed partial class PackageDetailsSettingsCard : SettingsCard

```

Comment:
Nit: eventually I would like to update our stylecop version, and one thing it complains about is no newline here. So I'm trying to get them into new files.
```suggestion
namespace DevHome.SetupFlow.Controls;

public sealed partial class PackageDetailsSettingsCard : SettingsCard
```